tests/test_pxssh.py

```diff
@@ -48,6 +48,15 @@ def test_failed_set_unique_prompt(self):
         else:
             assert False, 'should have raised exception, pxssh.ExceptionPxssh'
 
+    def test_connection_refused(self):
+        ssh = pxssh.pxssh()
+        try:
+            ssh.login('noserver', 'me', password='s3cret')
+        except pxssh.ExceptionPxssh:
+            pass
+        else:
+            assert False, 'should have raised exception, pxssh.ExceptionPxssh'
```

Comment:
This can be written more simply as
<pre>
ssh = pxssh.pxssh()
with self.assertRaises(pxssh.ExceptionPxssh):
    ssh.login('noserver', 'me', password='s3cret')
</pre>